tests/pfcwd/test_pfcwd_warm_reboot.py

```diff
@@ -624,6 +627,10 @@ def test_pfcwd_wb(self, fake_storm, testcase_action, setup_pfc_test, enum_fanout
             pytest.skip("Test skipped: No neighbors detected as 'rx_port_id' is None for selected test ports,"
                         " which is necessary for PFCwd test setup.")
 
+        if testcase_action != "no_storm":
```

Comment:
Do we plan to skip the case on all platforms or only on some special platforms?

Reply:
It's not supported on platforms which use ZeroBuffer in PfcWd, will update PR and limit the skip on cisco-8000 for now.

Reply:
If it's not supported on specific platform, maybe we could skip it in mark conditions, I don't think it's necessary to run the FPCWD warm reboot case with only the no_storm parameter

Reply:
Updated. Skip test_pfcwd_warm_reboot.py on cisco-8000 platform in tests_mark_conditions.yaml.